src/utils/fetch-config.ts

```diff
@@ -11,11 +11,19 @@ export async function fetchConfig (): Promise<void> {
   const processEnvBaseUrl: string = process.env.BASE_URL // eg, /business/
   const windowLocationPathname = window.location.pathname // eg, /business/CP1234567/...
   const windowLocationOrigin = window.location.origin // eg, http://localhost:8080
+  const windowLocationSearch = window.location.search // eg, ?accountid=2288
 
   if (!processEnvBaseUrl || !windowLocationPathname || !windowLocationOrigin) {
     return Promise.reject(new Error('Missing environment variables'))
   }
 
+  // get and store account id, if present
+  const accountId = new URLSearchParams(windowLocationSearch).get('accountid')
+  if (accountId) {
+    sessionStorage.setItem('ACCOUNT_ID', accountId)
+    console.log('Set Account ID to: ' + accountId)
+  }
```

Comment:
Note that SbcHeader has code to strip "accountid=xxx" out of the URL (so you don't see it) but THIS code works fine because it's run before SbcHeader is created.